Source/Scene/Cesium3DTileset.js

```diff
@@ -898,6 +902,15 @@ function Cesium3DTileset(options) {
    */
   this.debugShowUrl = defaultValue(options.debugShowUrl, false);
 
+  /**
+   * Function for examining vector lines as they are being streamed.
+   *
+   * @type {Function}
+   * @default undefined
+   * @experimental This feature is experimental and is subject to change without Cesium's standard deprecation policy.
```

Comment:
CC https://github.com/CesiumGS/cesium/pull/9398#discussion_r585836031 I'm not sure if it's worth making this experimental since the callback signature isn't documented and one of the parameters is a `Cesium3DTileBatchTable` which isn't in the public API.

I would just keep this private and remove `examineVectorLinesFunction` from the constructor's doc as well.